cats_frontend/src/app/modules/cats/pages/cats-list/cats-list.component.ts

```diff
@@ -59,144 +61,117 @@ export class CatsListComponent implements OnInit {
         description: new FormControl('', [Validators.required])
     });
 
+   
+
     constructor(
-        private catsService: CatsService,
```

Comment:
remove service injection from constructor, all the service have to be called in the effects